tests/bsim/bluetooth/host/central/src/main.c

```diff
@@ -8,9 +8,14 @@
 #include "babblekit/testcase.h"
 #include <zephyr/bluetooth/conn.h>
 
+/* Include conn_internal for the purpose of checking reference counts. */
+#include <zephyr/../../subsys/bluetooth/host/conn_internal.h>
```

Comment:
You should be able to just do 

```suggestion
#include "host/conn_internal.h"
```